src/components/ProjectsPage/__tests__/ProjectsPage.test.tsx

```diff
@@ -1,13 +1,48 @@
-import { render } from '@testing-library/react';
+import { act } from 'react-dom/test-utils';
```

Comment:
I would prefer using `@testing-library/react` over `react-dom/test-utils` for `act`

React team recommends the same https://testing-library.com/docs/react-testing-library/api#act